.github/workflows/lint-secrets.yml

```diff
@@ -18,21 +18,20 @@ jobs:
       # TODO(danh): this install+configure git-secrets is a good candidate for a
       # custom action repo -- closest to our needs I found is:
       # https://github.com/kams-mash/gh-secrets-scanner-action
+      - name: Set tempdir because github actions are dumb
+        run: echo "SECRETS_REPO=$(mktemp -dp "$PWD")" >> "$GITHUB_ENV"
```

Comment:
Can we put it in `/tmp` or something? Seems better not to have extra stuff hanging out in the main repo clone.

Reply:
Turns out, we can't. I initially tried to use the runner.tmpdir value which is guaranteed clean on each run, but actions/checkout refuses to put the repo outside of the workdir. I reasoned the rm already there was equally gross as any other workaround, so left it.

Reply:
Looks like I forgot to remove that bullet from the description, too. :)